src/locales/en.json

```diff
@@ -0,0 +1,28 @@
+{
+  "DEFAULT_ERRORS": {
```

Comment:
Lot of strings in this file are not being used.

Keep only that is being used.

Reply:
Deleted unwanted strings here.